20260619-en-google-misc-name (#2028)
utelecon/utelecon.github.io: src/pages/en/google/misc/name/index.mdx

  title: Changing your first and last name
---

import ExcuseForAccuracy from "@components/en/ExcuseForAccuracy.mdx";

<ExcuseForAccuracy />

## Overview
{:#overview}

> - The “profile name” of your ECCS Cloud Email account (shown as “Name” on the screen) is automatically created through linkage from the system, but you can change it yourself.

Reviewer: ```suggestion
- The “profile name” of your ECCS Cloud Email account (shown as “Name” on the screen) is automatically created via system synchronization, but you can change it yourself.
```

- In some systems, the [“system-managed name” may be displayed instead of your profile name](#system-name-issue). Note that users cannot change their “system-managed name”.

## How to change your profile’s first and last name
{:#edit-name}

> 1. In your browser, [log in to ECCS Cloud Email](/google/#login).

Reviewer: ```suggestion
1. In your browser, [log in to ECCS Cloud Email](../../#login).
```
